### PR TITLE
Add HockeyApp crash report documentation process. 

### PR DESCRIPTION
**Pull request by eschlange**
[#113454719]

### CODE REVIEW

File: tools/hockeyapp/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## Exception documentation process
+
+In order to capture and use all of the information gained from the HockeyApp crash
+reports, follow these steps upon notification:
+
+1. Create a Pivotal bug story for the issue.
+1. Add details to the bug story with a summary, context, and a URL to the HockeyApp
+   crash group.
+1. Finally annotate the HockeyApp crash group with a link to the Pivotal bug.
+1. After the bug is fixed and the Pivotal story is delivered, mark the HockeyApp
```
Comment (ericcf) [inline tools/hockeyapp/README.md]: Maybe change this slightly to indicate that the crash group should be marked resolved after the app fix has been distributed.

**Comment by ericcf** [comment]
Would you mind adding a link from the top level README to this page? Thanks.

File: tools/hockeyapp/README.md (hunk 1 above)
Comment (eschlange) [inline tools/hockeyapp/README.md]: Added more detail to include your recommendation.

**Comment by eschlange** [comment]
Added link from home page.

**Comment by ericcf** [comment]
:coffee:

**Pull request merged**